vividus-plugin-excel/src/main/java/org/vividus/excel/ExcelSheetParser.java

```diff
@@ -225,6 +227,23 @@ public String getDataFromCell(String address)
                 .orElse(null);
     }
 
+    public Map<String, List<String>> exactDataTable(String range)
+    {
+        CellRangeAddress address = CellRangeAddress.valueOf(range);
+        Map<String, List<String>> mapData = new HashMap<>();
+        for (int colIndex = address.getFirstColumn(); colIndex <= address.getLastColumn(); colIndex++)
+        {
+            List<String> temp = new ArrayList<>();
+            for (int rowIndex = address.getFirstRow() + 1; rowIndex <= address.getLastRow(); rowIndex++)
+            {
+                Row row = sheet.getRow(rowIndex);
+                temp.add(row.getCell(colIndex).getStringCellValue());
+            }
```

Comment:
if you need to get data from cols with indexes like 1,5,6 they you will get row with index 1,2,3..... for each of those col indexes, so can we iterate through rows and populate cols without having to access the same rows multiple times?

Reply:
Fix this